kratos/utilities/binbased_fast_point_locator.h

```diff
@@ -131,7 +131,9 @@ class BinBasedFastPointLocator
                          array_1d<double, TDim + 1 > & N,
                          Element::Pointer& pelement,
                          ResultIteratorType result_begin,
-                         const unsigned int MaxNumberOfResults)
+                         const unsigned int MaxNumberOfResults,
+                         const double Tolerance = 1.0e-3
```

Comment:
@pooyan-dadvand , like I merged I created a new pull request #65, the discussion should be there